lib/rdf2marc/rdf2model/mappers/leader.rb

```diff
@@ -17,6 +17,22 @@ def generate
 
         private
 
+        def record_status
+          item.admin_metadata.query.path_first_literal([[BF.status, BF.Status], BF.code])
+        rescue Rdf2marc::MappingError => e
+          return default_record_status if status_not_literal_pattern.match?(e.message)
+
+          raise e
```

Comment:
The code that was there previously could (and did!) raise errors, and was uncovered then. It remains uncovered now, with new testing added for the specific added condition on line 23.